### DISCUSSION TITLE
Icon request: jail

### DISCUSSION THREAD

**Discussion by Vringe** [FortAwesome/Font-Awesome#21163] [Icon Requests]
### What can it be used for?

Can be used in a game to put a player in jail.

### Example image (optional)
![Jail Icon](https://image.flaticon.com/icons/png/512/125/125498.png)

*Source: https://www.flaticon.com/authors/freepik*

### Icon request checklist

- [x] This is a single icon or matched pair (Ex: `lock` / `unlock`)
- [x] The title starts with "Icon request: " and is followed by the requested icon name (Ex: `Icon request: magical-unicorn`)
- [x] I have [searched for existing issues](https://github.com/FortAwesome/Font-Awesome/issues) and to the best of my knowledge this is not a duplicate
- [x] I have included at least one example image if it could be helpful (optional)
- [x] The request is for a concrete object (it's harder to make an icon to represent happiness, it's easier to make a smiley face. ☺)

**Comment by TexasSwede** [comment]
I could use it for a system I am currently building for lawyers.

**Comment by briciou62500** [comment]
this icon to you was created?

**Comment by sensibleworld** [comment]
This is a quick update to let folks know that we’re refining our icon request workflow behind the scenes, and as part of that we’re going through all open requests and filing them into several new buckets to be addressed.

In this case we’ve added this issue to our new list to be created! We can’t promise any firm timelines right now, but this will help us keep track of it.

We’ll update this issue when the icon is released into the wild 🙂
